mlrun/datastore/datastore_profile.py

```diff
@@ -185,6 +185,17 @@ class DatastoreProfileS3(DatastoreProfile):
     assume_role_arn: typing.Optional[str] = None
     access_key_id: typing.Optional[str] = None
     secret_key: typing.Optional[str] = None
+    bucket: typing.Optional[str] = None
+
+    @pydantic.validator("bucket")
+    def check_bucket(cls, v):
+        if v is None:
```

Comment:
```suggestion
        if not v:
```
I'm assuming an empty string won't cut it either.
(all instances)